src/features/governance/RevokeProxyScreen.tsx

```diff
@@ -0,0 +1,269 @@
+import BackScreen from '@components/BackScreen'
+import Box from '@components/Box'
+import ButtonPressable from '@components/ButtonPressable'
+import CircleLoader from '@components/CircleLoader'
+import { Select } from '@components/Select'
+import Text from '@components/Text'
+import {
+  batchInstructionsToTxsWithPriorityFee,
+  bulkSendTransactions,
+  populateMissingDraftInfo,
+  toVersionedTx,
+  truthy,
+} from '@helium/spl-utils'
+import {
+  PositionWithMeta,
+  useUnassignProxies,
+} from '@helium/voter-stake-registry-hooks'
+import { usePublicKey } from '@hooks/usePublicKey'
+import { RouteProp, useNavigation, useRoute } from '@react-navigation/native'
+import { PublicKey, TransactionInstruction } from '@solana/web3.js'
+import { useGovernance } from '@storage/GovernanceProvider'
+import { MAX_TRANSACTIONS_PER_SIGNATURE_BATCH } from '@utils/constants'
+import sleep from '@utils/sleep'
+import { getBasePriorityFee } from '@utils/walletApiV2'
+import React, { useCallback, useMemo, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { FlatList } from 'react-native'
+import { Edge } from 'react-native-safe-area-context'
+import { useSolana } from '../../solana/SolanaProvider'
+import { useWalletSign } from '../../solana/WalletSignProvider'
+import { WalletStandardMessageTypes } from '../../solana/walletSignBottomSheetTypes'
+import { PositionPreview } from './PositionPreview'
+import { ProxySearch } from './ProxySearch'
+import { GovernanceStackParamList } from './governanceTypes'
+
+type Route = RouteProp<GovernanceStackParamList, 'RevokeProxyScreen'>
+
+export const RevokeProxyScreen = () => {
+  const route = useRoute<Route>()
+  const { wallet, position } = route.params
+  const { t } = useTranslation()
+  const [proxyWallet, setProxyWallet] = useState(wallet)
+  const proxyWalletKey = usePublicKey(proxyWallet)
+  const positionKey = usePublicKey(position)
+  const { positions, refetch } = useGovernance()
+
+  const networks = useMemo(() => {
+    return [
+      { label: 'HNT', value: 'hnt' },
+      { label: 'MOBILE', value: 'mobile' },
+      { label: 'IOT', value: 'iot' },
+    ]
+  }, [])
+  const [network, setNetwork] = useState('hnt')
```

Comment:
Think i've seen this multiple times in this code. `useGovernance()` comes with a `network` of type 

```typescript
enum GovNetwork {
  hnt = 'Helium',
  mobile = 'Helium MOBILE',
  iot = 'Helium IOT',
}
```

and `setMint` that takes a `publicKey`. Wonder if we can tie into that for this logic?